pkg/icassigner/busyness/busyness.go

```diff
@@ -158,27 +154,47 @@ func (b *githubBusynessClient) getBusyness(ctx context.Context, since time.Time,
 	// count relevant issues
 	busyness := 0
 	for _, i := range issues {
-		// ignore everything without any state
-		if i.State == nil {
-			continue
-		}
-
-		switch *i.State {
+		switch i.GetState() {
 		case "open":
 			// check for labels to ignore, e.g. `stale` and ignore issue in this case
-			if containsLabelsToIgnore(i.Labels) {
+			if b.containsLabelsToIgnore(i.Labels) {
+				log.Printf("%s: Ignoring open issue because it contains labels to ignore (%s): %s\n", member, labelsToString(i.Labels), i.GetTitle())
 				continue
 			}
 
 			// increase busyness count otherwise
+			log.Printf("%s: Issue increases busyiness because it is still open: %s\n", member, i.GetTitle())
 			busyness++
 		case "closed":
 			// if the issue got closed since our time to check
-			if since.Before(*i.ClosedAt) {
+			if since.Before(i.GetClosedAt()) {
+				log.Printf("%s: Issue increases busyiness because it has been closed at %s which is after %s: %s\n", member, i.GetClosedAt().String(), since.String(), i.GetTitle())
```

Comment:
```suggestion
				log.Printf("%s: Issue increases busyness because it has been closed at %s which is after %s: %s\n", member, i.GetClosedAt().String(), since.String(), i.GetTitle())
```